File: 1.FDP/2° Competencia/Java/Seleccion/Ejercicio2.java
import java.util.Scanner;	

public class Ejercicio2 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		//
		System.out.println("Calcular el area de un rectagulo.\n");
		double Base, Altura, Area;
		System.out.println("Base: ");
		Base= scan.nextDouble();
		if (Base > 0) {
			System.out.println("Altura: ");
			Altura= scan.nextDouble();
			if (Altura > 0) {
					Area= Base*Altura;
				System.out.println("Area= " + Area);
			} else {
				System.out.println("Error en la altura");
			}
		} else {
			System.out.println("Error en la base");
		}
		//
	}
}

File: 1.FDP/2° Competencia/C++/Seleccion/Ejercicio2.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
	cout << "Calcular el area de un rectangulo.\n";
	double Altura, Base, Area;
	cout << "Ingresar la base:";
	cin >> Base;
	if (Base > 0){
		cout << "Ingresar la altura:";
		cin >> Altura;
		if (Altura > 0) {
			Area= Altura*Base;
			cout << "Area = " << Area << "\n";
		} else {
			cout << "Error en la altura.\n";
			
		}
	} else {
		cout << "Eror en la base.\n";
		
	}
	
	return 0;
}
